### PR TITLE
Use "requirement" in grammar font in expr.prim.req

### PR DESCRIPTION
**Pull request by akrzemi1**
[expr.prim.req] uses grammarterm <em><code>requirement</code></em> in paragraph 5, but normal font in paragraph 6. Term "requirement" is never defined in this clause, but it has a different meaning in the library.

### CODE REVIEW

File: source/expressions.tex [hunk 1]
```diff
@@ -2567,7 +2567,7 @@
 \pnum
 The substitution of template arguments into a \grammarterm{requires-expression}
 may result in the formation of invalid types or expressions in its
-requirements or the violation of the semantic constraints of those requirements.
+\grammarterm{requirement}{}s or the violation of the semantic constraints of those \grammarterm{requirement}{}s.
```
Comment (jensmaurer) [inline source/expressions.tex]: Please move the "s" inside the empty brace immediately preceding it.

File: source/expressions.tex [hunk 2]
```diff
@@ -2577,7 +2577,7 @@
 the \grammarterm{requires-expression} evaluates to \tcode{true}.
 \begin{note}
 If a \grammarterm{requires-expression} contains invalid types or expressions in
-its requirements, and it does not appear within the declaration of a templated
+its \grammarterm{requirement}{}s, and it does not appear within the declaration of a templated
```
Comment (jensmaurer) [inline source/expressions.tex]: Also here.

**Comment by jensmaurer** [comment]
You change some "s" in places that weren't previously in scope for this pull request (ok), but you didn't change the "s" things for the stuff that you originally changed here. That's not really progress.

**Comment by akrzemi1** [comment]
Sorry. Learning git pull requests and tex at the same time...

**Comment by zygoloid** [comment]
There's no need for braces around or near the `s`s at all. Will remove those now...

**Comment by jensmaurer** [comment]
(Braces were required in former times due to LaTeX-isms.)

**Pull request merged**